### DISCUSSION TITLE
Cool Project

### DISCUSSION THREAD

**Discussion by suryathink** [Shubhamtribhuvan8/real-time-chat-app#3] [General]
How did you build this?

This is so awesome project. 

The feature of Private messaging and message History is very good.

Really Liked it.

**Comment by Shubhamtribhuvan8** [comment]
@suryathink **I used Firebase, Node.js, and React.js for this project, and deployed it using Firebase. Thank you for sharing these details, and I appreciate your kind words about the project. If you `have` any further questions or need assistance, please feel free to ask.**
